models/cart-discount-value-gift-line-item/src/cart-discount-value-gift-line-item-draft/generator.ts

```diff
@@ -0,0 +1,17 @@
+import { Reference } from '@commercetools-test-data/commons';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { TCartDiscountValueGiftLineItemDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/cartDiscounts#cartdiscountvaluegiftlineitemdraft
```

Comment:
🏆 and the award for longest type name goes to....

Reply:
Omg tell me about it